### ISSUE TITLE
[Usable – Make Code - Blocks] focus moves to disable control and not announce as disable by screen reader.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3631]
User Experience:
Users who depends upon screen reader get confused if the Screen reader reads does not announce disable for disable control when focus moves to disable control.

Note: User credentials should NOT be included in the bug.

Repro Steps:

Launch application URL
Home page will open.
Navigate "New Project" option and select it.
Editor screen will open.
Navigate to "Simulator" section appearing in left pane.
Navigate to "Toggle Debug Mode" option and select it.
Navigate through all newly appeared controls like Play/pause, Exit debug mode, Step over, step into, step out etc.
Verify the issue.

Actual Result:
focus moves to disable control and not announce as disable by screen reader.

Expected Result:
If focus moves to disable control then screen reader should announce as disable for those control.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Uploading Usable_Disable control.zip…]()
